src/modules/web3/web3.command.ts

```diff
@@ -0,0 +1,36 @@
+import {Command} from 'djs-slash-helper'
+import {ApplicationCommandType} from 'discord-api-types/v10'
+import {CommandInteraction, EmbedBuilder} from 'discord.js'
+import Parser from 'rss-parser'
+import TurndownService from 'turndown'
+
+
+const web3RssUrl = 'https://web3isgoinggreat.com/feed.xml'
+
+const rssParser = new Parser()
+
+const turndownService = new TurndownService({ bulletListMarker: '-' })
+
+export const Web3Command: Command<ApplicationCommandType.ChatInput> = {
+	name: 'web3fact',
```

Comment:
is calling it "web3**fact**" a good idea? the source is truthful but very biased, i can see how that may cause contention

Reply:
It gives you a fact about Web3 :+1: hope this helps